check-disk/lib/check-disk.go

```diff
@@ -0,0 +1,212 @@
+package checkdisk
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/mackerelio/checkers"
+	gpud "github.com/shirou/gopsutil/disk"
+)
+
+var opts struct {
```

Comment:
[Suggestion]
Original check_disk has so many options. I don't think check-disk must implement all of them, but some of them are not hard to implement, so please consider it. (Or you can do it in another p-r.)
- inode monitoring threshold `-W`/`-K`
- fstype filtering `-X`/`-N`
- Explicit check against all partitions `-A`